.github/workflows/build-release.yml

```diff
@@ -111,3 +111,22 @@ jobs:
           files: |
             build/flow.phar
             build/flow.phar.asc
+
+      - name: "Get version"
+        id: get_version
+        run: |
+          echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_OUTPUT
```

Comment:
I think you should be able to use `$BUILD_TAG` that is extracted one step before this one.

Reply:
Tag doesn't contain dots, which leads to broken URL:
```console
Run mislav/bump-homebrew-formula-action@v3
  with:
    formula-name: flow-php
    formula-path: src/tools/homebrew/Formula/flow-php.rb
    homebrew-tap: stloyd/flow
    base-branch: feature/homebrew
    download-url: https://github.com/stloyd/flow/releases/download/054/flow.phar
    commit-message: {{formulaName}} {{version}}
  
  Created by https://github.com/mislav/bump-homebrew-formula-action
  
  env:
    COMPOSER_PROCESS_TIMEOUT: 0
    COMPOSER_NO_INTERACTION: 1
    COMPOSER_NO_AUDIT: 1
    BUILD_TAG: 054
    COMMITTER_TOKEN: ***
GET /repos/stloyd/flow/releases/tags/054 - 404 in 311ms
```

Reply:
In this case, I would move definitions of those two variables to the beginning of the workflow and add some comments about them.

Reply:
Fixed that build tag (renamed it too) as it's some leftover from past, and used in new workflow.